ui/src/common/components/CitationSummaryGraph/CitationSummaryGraph.jsx

```diff
@@ -138,6 +138,12 @@ class CitationSummaryGraph extends Component {
     return GRAY;
   }
 
+  static getCountLabel(docCount) {
+    if (docCount === 0) return null;
+    if (docCount < 10000) return docCount.toString();
```

Comment:
I know it isn't part of your task necessarily, but can you add ```<FormattedNumber>``` here? for cases under 10000. Discussed it with Stella.

Reply:
We're very limited in space in the graph so this makes the numbers overlap again in tablets and smaller screens, are we sure we need this here @StellaCh ? We only have 4 digit numbers so they're not hard to read.

Reply:
Well, we had this discussion before and we said that we should give it a try and see if it affects the display that much. Can you try and share the screenshots?

Reply:
So after discussing with Stella we're not going to add the separator here to avoid overlapping. I saw that it's missing in the table for h-index though, I added them here to avoid creating a PR just for that.